index.html

```diff
@@ -0,0 +1,347 @@
+                        fill="currentColor" />
+                </svg>
+            </a>
+        </div>
+
+        <div class="tools-summery">
+            <div class="header">
+                <h1>languages</h1>
+                <i class='bx bx-chevron-down'></i>
+            </div>
+
+
+            <div class="languages">
+                <div class="language">
+                    <img src="assets/images/image5.png" alt="">
+                    <p>javascript</p>
+                </div>
+
+                <div class="language">
+                    <img src="assets/images/image6.png" alt="">
+                    <p>html</p>
+                </div>
+
+                <div class="language">
+                    <img src="assets/images/css7.png" alt="">
+                    <p>css</p>
+                </div>
+            </div>
```

Comment:
- [ ] I noticed that the technology categories (e.g., "JavaScript," "HTML," and "CSS") are not in a list format. :wink:  :smile: 
To improve the readability of your project, I kindly recommand that you convert each technology category into an unordered list `ul` with each item as a list item `li`. Kindly fix this :+1: